addons/packages/package-overlay/package-overlay.yaml

```diff
@@ -0,0 +1,9 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+#@overlay/match by=overlay.subset({"kind":"Package"}),expects="1+"
+---
+spec:
+#@overlay/match missing_ok=True
+  valuesSchema:
+    openAPIv3:  #@ yaml.decode(data.values.openapi)["components"]["schemas"]["dataValues"]
```

Comment:
Should have ytt annotation like `@overlay/replace `. Otherwise, it will keep overlaying on the existing pakcage.yaml valuesSchema contents.

Reply:
Fixed